logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -72,6 +75,7 @@ internal fun UserSessionScope(
         userStorageProvider,
         clientConfig,
         platformUserStorageProperties,
-        networkStateObserver
+        networkStateObserver,
+        dataStore
```

Comment:
instead of using data store only on android target you can wrap datastore in a kmm settings and use is the same way we use shared pref from the common main code

Reply:
It's not for Android only, it supports iOS and JVM